.golangci.yaml

```diff
@@ -51,5 +51,9 @@ issues:
       text: "globalLogger.*unused"
       linters:
         - unused
+    - path: "^.*.go$"
+      text: "golang.org/x/crypto/openpgp.*deprecated"
+      linters:
+        - staticcheck
```

Comment:
Why do we need this?

Reply:
To suppress the `staticcheck` linting error:

```
internal/archive/testarchive/testarchive.go:11:2: SA1019: "golang.org/x/crypto/openpgp/clearsign" is deprecated: this package is unmaintained except for security fixes. New applications should consider a more focused, modern alternative to OpenPGP for their specific task. If you are required to interoperate with OpenPGP systems and need a maintained package, consider a community fork. See https://golang.org/issue/44226. (staticcheck)
	"golang.org/x/crypto/openpgp/clearsign"
	^
internal/archive/testarchive/testarchive.go:12:2: SA1019: "golang.org/x/crypto/openpgp/packet" is deprecated: this package is unmaintained except for security fixes. New applications should consider a more focused, modern alternative to OpenPGP for their specific task. If you are required to interoperate with OpenPGP systems and need a maintained package, consider a community fork. See https://golang.org/issue/44226. (staticcheck)
	"golang.org/x/crypto/openpgp/packet"
	^
internal/archive/archive.go:11:2: SA1019: "golang.org/x/crypto/openpgp/packet" is deprecated: this package is unmaintained except for security fixes. New applications should consider a more focused, modern alternative to OpenPGP for their specific task. If you are required to interoperate with OpenPGP systems and need a maintained package, consider a community fork. See https://golang.org/issue/44226. (staticcheck)
	"golang.org/x/crypto/openpgp/packet"
	^
internal/testutil/pgpkeys.go:6:2: SA1019: "golang.org/x/crypto/openpgp/packet" is deprecated: this package is unmaintained except for security fixes. New applications should consider a more focused, modern alternative to OpenPGP for their specific task. If you are required to interoperate with OpenPGP systems and need a maintained package, consider a community fork. See https://golang.org/issue/44226. (staticcheck)
	"golang.org/x/crypto/openpgp/packet"
	^
internal/pgputil/openpgp.go:8:2: SA1019: "golang.org/x/crypto/openpgp/armor" is deprecated: this package is unmaintained except for security fixes. New applications should consider a more focused, modern alternative to OpenPGP for their specific task. If you are required to interoperate with OpenPGP systems and need a maintained package, consider a community fork. See https://golang.org/issue/44226. (staticcheck)
	"golang.org/x/crypto/openpgp/armor"
	^
internal/pgputil/openpgp.go:9:2: SA1019: "golang.org/x/crypto/openpgp/clearsign" is deprecated: this package is unmaintained except for security fixes. New applications should consider a more focused, modern alternative to OpenPGP for their specific task. If you are required to interoperate with OpenPGP systems and need a maintained package, consider a community fork. See https://golang.org/issue/44226. (staticcheck)
	"golang.org/x/crypto/openpgp/clearsign"
	^
internal/pgputil/openpgp.go:10:2: SA1019: "golang.org/x/crypto/openpgp/packet" is deprecated: this package is unmaintained except for security fixes. New applications should consider a more focused, modern alternative to OpenPGP for their specific task. If you are required to interoperate with OpenPGP systems and need a maintained package, consider a community fork. See https://golang.org/issue/44226. (staticcheck)
	"golang.org/x/crypto/openpgp/packet"
	^
internal/setup/setup_test.go:8:2: SA1019: "golang.org/x/crypto/openpgp/packet" is deprecated: this package is unmaintained except for security fixes. New applications should consider a more focused, modern alternative to OpenPGP for their specific task. If you are required to interoperate with OpenPGP systems and need a maintained package, consider a community fork. See https://golang.org/issue/44226. (staticcheck)
	"golang.org/x/crypto/openpgp/packet"
	^
internal/pgputil/openpgp_test.go:4:2: SA1019: "golang.org/x/crypto/openpgp/packet" is deprecated: this package is unmaintained except for security fixes. New applications should consider a more focused, modern alternative to OpenPGP for their specific task. If you are required to interoperate with OpenPGP systems and need a maintained package, consider a community fork. See https://golang.org/issue/44226. (staticcheck)
	"golang.org/x/crypto/openpgp/packet"
	^
internal/setup/setup.go:12:2: SA1019: "golang.org/x/crypto/openpgp/packet" is deprecated: this package is unmaintained except for security fixes. New applications should consider a more focused, modern alternative to OpenPGP for their specific task. If you are required to interoperate with OpenPGP systems and need a maintained package, consider a community fork. See https://golang.org/issue/44226. (staticcheck)
	"golang.org/x/crypto/openpgp/packet"
	^
internal/archive/archive_test.go:4:2: SA1019: "golang.org/x/crypto/openpgp/packet" is deprecated: this package is unmaintained except for security fixes. New applications should consider a more focused, modern alternative to OpenPGP for their specific task. If you are required to interoperate with OpenPGP systems and need a maintained package, consider a community fork. See https://golang.org/issue/44226. (staticcheck)
	"golang.org/x/crypto/openpgp/packet"
	^
```

Reply:
Thanks! Only one request, can we change the regexp to be as specific as possible? Something like:
```
\"golang.org/x/crypto/openpgp/\w+\" is deprecated
```

Reply:
Updated, thanks! https://github.com/canonical/chisel/pull/130/commits/4dbee41d64c46077528f26482c7a7333b2729b09